app/script/entity/message/Message.js

```diff
@@ -202,6 +202,12 @@ z.entity.Message = class Message {
     );
   }
 
+  isMention() {
+    return (
+      this.has_asset_text() && this.assets().some(assetEntity => assetEntity.is_text() && assetEntity.mentions().length)
```

Comment:
Only self mentions are counted. There is already a `isSelfMentioned()` in place on the message to take care of that. You can add the some assets check there since it potentially fails in my code I belive.

Reply:
Yes, thanks, I just noticed, too 🤦‍♂️